indra/llui/lllineeditor.h

```diff
@@ -179,6 +179,7 @@ class LLLineEditor
 	virtual void	onFocusReceived();
 	virtual void	onFocusLost();
 	virtual void	setEnabled(BOOL enabled);
+	/*virtual*/ const std::string getToolTip() const override;
```

Comment:
Either not 'override' this function, or 'override' all of them, otherwise mac won't be happy.